VisualStudio.gitignore

```diff
@@ -205,6 +205,9 @@ orleans.codegen.cs
 # (https://github.com/github/gitignore/pull/1529#issuecomment-104372622)
 #bower_components/
 
+# VS15: by default bower location is moved, bower restore is true
+**/wwwroot/lib/
```

Comment:
On line 29 we have this rule:

```
# Uncomment if you have tasks that create the project's static files in wwwroot
#wwwroot/
```

This was added in https://github.com/github/gitignore/pull/1590 and feels familiar to this. Could you:

 - clarify what's in `wwwroot/lib` on your machine so we can document here what we expect (unless there are some external resources to point to) 
 - see if we can update that rule to reflect whatever changes are coming in VS2017